packages/react/src/analytics/integrations/Forter/Forter.ts

```diff
@@ -0,0 +1,192 @@
+   * Needs to wait for a user to be set in analytics by analytics.setUser method
+   * in order to have the `customerId` parameter filled as it is a required
+   * parameter by Omnitracking.
+   *
+   * @param forterToken - String representing the loaded forter token.
+   * @returns Promise that will be resolved when the user is set in analytics and the Omnitracking message is built.
+   *
+   */
+  async sendForterTokenLoadedEventForOmnitracking(forterToken: string) {
+    await this.onSetUserPromise;
+
+    const omnitrackingMessage = await this.getForterTokenLoadedEventPayload(
+      forterToken,
+    );
+
+    if (this.options.omnitrackingHttpClient) {
+      await this.options.omnitrackingHttpClient(omnitrackingMessage);
+      return;
+    }
+    await postTracking({ ...omnitrackingMessage });
+  }
+
+  /**
+   * Generates an Omnitracking message as specified by the documentation
+   * in order to be posted to Omnitracking and thus, obtain the reference
+   * to the forter token for this session.
+   *
+   * @param forterToken - String representing the loaded forter token.
+   *
+   * @returns Promise that will be resolved when the Omnitracking message is built.
+   */
+  async getForterTokenLoadedEventPayload(forterToken: string) {
+    const forterTokenLoadedEventData =
+      await this.createSyntheticForterTokenLoadedEvent();
+
+    const additionalParameters = {
+      tid: ForterTokenTid,
+      val: JSON.stringify({
+        forterTokenCookie: forterToken,
+        origin: get(this.options, 'origin', 'NOT_DEFINED'),
+        userAgent: get(
+          forterTokenLoadedEventData,
+          'context.web.window.navigator.userAgent',
+        ),
+      }),
+    };
+
+    const payload = integrations.Omnitracking.formatTrackEvent(
+      forterTokenLoadedEventData,
+      additionalParameters,
+    );
+
+    return payload;
+  }
+
+  /**
+   * Sets up the promise that will be resolved when onSetUser of this instance
+   * is called by analytics.
+   */
+  setupOnSetUserPromise() {
+    this.onSetUserPromise = new Promise<SetUserEventData>(resolve => {
+      this.resolveOnSetUserPromise = resolve;
+    });
+  }
+
+  /**
+   * Loads the forter script with the corresponding siteId.
+   *
+   * @param siteId - Site identifier required by forter script and is configured by the user when adding this integration with `analytics.addIntegration` method.
+   */
+  loadForterScript(siteId: string) {
+    loadForterScriptForSiteId(siteId);
+  }
+
+  /**
+   * Resolves the configured set user promise.
+   */
+  override onSetUser(data: SetUserEventData) {
+    if (this.resolveOnSetUserPromise) {
+      this.resolveOnSetUserPromise(data);
+      this.resolveOnSetUserPromise = null;
+    }
+  }
+
+  /**
+   * Tracks events (page views or page actions).
+   *
+   * @param _data - The event object.
+   *
+   */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  override track(_data: EventData<TrackTypesValues>) {
```

Comment:
```suggestion
  override track() {
```

With this, you can safely remove the `eslint-disable-next-line` rule